wallet/txauthor/go.sum

```diff
@@ -22,8 +22,6 @@ github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f/go.mod h1:TdznJufo
 github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d/go.mod h1:+5NJ2+qvTyV9exUAL/rxXi3DcLg2Ts+ymUAY5y4NvMg=
 github.com/btcsuite/btcwallet/wallet/txrules v1.2.0 h1:BtEN5Empw62/RVnZ0VcJaVtVlBijnLlJY+dwjAye2Bg=
 github.com/btcsuite/btcwallet/wallet/txrules v1.2.0/go.mod h1:AtkqiL7ccKWxuLYtZm8Bu8G6q82w4yIZdgq6riy60z0=
-github.com/btcsuite/btcwallet/wallet/txsizes v1.2.2 h1:vtfutRuUoNsdgmKgYShuUgmBawNeC+lWu9jeucWkAII=
```

Comment:
What version of Go did you run this on? I'm getting a diff in `wallet/{txauthor,txrules,txsizes}` with Go 1.18 or later.

Reply:
mine is `go version go1.20.5 darwin/arm64`. What are the diffs here? I think maybe it's due to my local go work setup, that it also makes sure this repo is in sync with `lnd`.

Reply:
Here's the diff I get when running `go mod tidy`:
[diff.txt](https://github.com/btcsuite/btcwallet/files/12730000/diff.txt)

Reply:
yeah very weird, for instance this line
```
github.com/golang/protobuf v1.5.2/go.mod h1:XVQd3VNwM+JqD3oG2Ue2ip4fOMUkwXdXDdiuN0vRsmY=
```

looks like this is in sync with lnd's mod which also has this version.